core/metacling/src/TClingRdictModuleFileExtension.cxx

```diff
@@ -123,7 +123,9 @@ TClingRdictModuleFileExtension::Reader::Reader(clang::ModuleFileExtension *Ext,
       }
       case FIRST_EXTENSION_RECORD_ID + 1: {
          // FIXME: Remove the string copy in fPendingRdicts.
-         llvm::SmallString<255> FullRdictName = llvm::sys::path::parent_path(Mod.FileName);
+         llvm::StringRef ModDir = llvm::sys::path::parent_path(Mod.FileName);
```

Comment:
@vgvassilev , this still does not work. You need to ResolveSymlink for Mod.FileName and then get its directory. Note the Mod.FileName itself is symlink.

Reply:
That exactly what I meant when typing this code. Thanks for spotting it!